source/projectile/Projectile.hx

```diff
@@ -0,0 +1,33 @@
+package projectile;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.math.FlxPoint;
+
+class Projectile extends FlxSprite {
+	private static var SPEED:Float = 0;
+
+	public function new() {
+		super(FlxG.width, FlxG.height);
+		kill();
+	}
+
+	override public function update(elapsed:Float) {
+		super.update(elapsed);
+	}
```

Comment:
There's no need to override `update()` if you don't have any new logic inside - by default calling its update function will call its parent's update method.